### PR TITLE
Shell files

### PR DESCRIPTION
**Pull request by parthvshah**
I have created a folder called "scripts" in the titanium-silver folder. This can be easily changed to the "codes" folder. Please have a look and let me know if this is what you were looking for.

### CODE REVIEW

File: titanium_silver/c_lang.py [hunk 1]
```diff
@@ -2,6 +2,12 @@
 from titanium_silver.lang_container import LangContainer 
 import pdb
 
+def read_cmd():
+    cmd_fd = open('./scripts/c_lang.sh', 'r')
```
Comment (ganesh-k13) [inline titanium_silver/c_lang.py]: Please use:
`with open('<your file>'):`

File: titanium_silver/c_lang.py [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+    cmd = cmd_fd.readlines()
```
Comment (ganesh-k13) [inline titanium_silver/c_lang.py]: just do return `return cmd_fd.read()`

File: titanium_silver/scripts/c_lang.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+sh -c gcc /opt/%s.c -o /opt/%s && /opt/%s %s < %s
```
Comment (ganesh-k13) [inline titanium_silver/scripts/c_lang.sh]: Add a shebang: `#!/bin/bash`

**Comment by ganesh-k13** [comment]
Please follow this: https://stackoverflow.com/a/5601839/5671364 for package data.

It may take time but will avoid a lot of future troubles. Good idea on the scripts being part of the package but follow paths as shown in the link

Those are all the changes, please mail me here for queries: ganesh3597@gmail.com (I can reply my number here if needed)

Thanks and Regards,
Ganesh K.

**Comment by parthvshah** [comment]
Hey Ganesh,

Thanks a lot for the reviews. I'll look into it and get back to you.

Regards,
-Parth Shah
parthvshah.me

**Comment by ganesh-k13** [comment]
Hey bro, I'm traveling a few days, the Travis Builds are failing so have a look,
So to run tests locally: 
1. `pytest` for all
2. `pytest tests/test_c.py` for individual (recommend)

**Comment by ganesh-k13** [comment]
Is someone following up on this?

**Comment by parthvshah** [comment]
Aniket will be sending in a PR. He's in charge of this integration now.

**Pull request closed without merging**